view/utils/tablesorter-time-range-comarator.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+function contains(a, searchString) {
```

Comment:
Basically this whole module looks like a lot of code do what is needed.

I propose something quite simpler:

```javascript
var getTime = function (res) {
	var time = 0; // in seconds
	if (res.indexOf('<') !== -1) {
		return -1;
	}
	if (res.indexOf('h') !== -1) {
		res = res.split('h');
		time += Number(res[0]) * 3600;
		res = res[1];
	}
	if (res.indexOf('m') !== -1) {
		time += Number(res.split('m')[0]) * 60;
	}

	return time;
};

module.exports = function (a, b) {
	var resA, resB, timeA = 0, timeB = 0;
	resA = a.replace(/\s+/g, '');
	resB = b.replace(/\s+/g, '');

	timeA = getTime(resA);
	timeB = getTime(resB);
	
	return timeA - timeB;
};
```

This sorts ascending....